components/Link.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import NextLink from 'next/link'
```

Comment:
From the docs it is `import Link from 'next/link'`?

Reply:
if I `import Link from 'next/link' I get a parsing error 
![image](https://user-images.githubusercontent.com/24685932/212588552-7f5278ee-4953-4aa9-ad26-541ec03bd302.png)

I'm just creating an alias out of the Link to get rid of that parsing error. See docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#default_import

> Since the default export doesn't explicitly specify a name, you can give the identifier any name you like.

Reply:
I am exporting it as an alias to avoid a parsing error. 
![image](https://user-images.githubusercontent.com/24685932/212592826-d8911d27-a741-4df9-b3e9-17a24ccdf471.png)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#default_import

Reply:
Ah I see, thank you for clarifying 👍 

Hmm maybe changing the name of our component would be more clear, great idea 💯 

Now what to call it? 🤔

Reply:
I'll throw out a few names, I'm unsure about what the name should be too. I almost like the approach of aliasing the next Link because a developer might search for a Link component, but maybe not a Custom/Hyper/Styled-Link.

CustomLink?
HyperLink?
StyledLink?

I'm definitely open to suggestions for names too.

Reply:
`StyledLink` looks good to me 👍 - maybe let's go with this and we can always rename it again later on if someone can come up with a better name

Reply:
What do you think?